src/mixins.ts

```diff
@@ -27,10 +28,7 @@ export default {
       return numeral(number).format(format);
     },
     _shorten: shorten,
-    _ipfsUrl(ipfsHash: string): string | null {
-      if (!ipfsHash) return null;
-      return `https://${process.env.VUE_APP_IPFS_GATEWAY}/ipfs/${ipfsHash}`;
-    },
+    _ipfsUrl: getUrl,
```

Comment:
We could rename the function too to be consistent
```suggestion
    _getUrl: getUrl,
```